game-app/game-core/src/test/java/games/strategy/triplea/ui/panels/map/MapRouteDrawerTest.java

```diff
@@ -137,7 +137,15 @@ void verifySequenceIsTrulyMonotonic(final Point2D[] points) {
             Arguments.of(new Double(1, 1)),
             Arguments.of(new Double(-1, -1)),
             Arguments.of(new Double(1, -1), new Double(1, -1), new Double(1, -1)),
-            Arguments.of(new Double(-1, -1), new Double(1, 1), new Double(0, 0)))
+            Arguments.of(new Double(-1, -1), new Double(1, 1), new Double(0, 0)),
+            // 13.44 is chosen specifically to test a rare edge case that used to cause an exception
```

Comment:
The commentary is improved, but we still have a lot of it. This is quite a lot to read and understand & maintain. Perhaps we can still do better. Why is 13.44 all that special? Are there other special values? What is the range of valid values accepted by the computation function? What if we simply test nearly all of them with an increment of 0.01? Where would that land us with the runtime of the test case?

Reply:
Ok, I did some experiments and it seems like 13.44 is not all that special because this bug occurs with almost every "large" positive integer.
So I changed the test case to use '1' als a number instead which tests the same thing and adjusted the comment accordingly.